.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: weekly
+      day: saturday
```

Comment:
nit: make it a weekday so my notification inbox isn't flooded monday morning

Reply:
But then CI is filled
If you're reactive to merge things, putting it to daily makes it smoother. But you have to merge things and not go stale :shrug: https://github.com/nextcloud/nextcloud-cypress/blob/master/.github/dependabot.yml

Reply:
At night it's filled as well?

You can leave it at Saturday if you prefer that.